### DISCUSSION TITLE
Hi Prisma Team, is there any plan to fix the dbgenerated bug?

### DISCUSSION THREAD

**Discussion by Akumzy** [prisma/prisma#24662] [Q&A]
### Question

This bug [here](https://github.com/prisma/prisma/issues/2058) is pushing us away from using `dbgenerated` for default values

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @Akumzy 👋 

The link you shared does not point to the dbgenerated bug you mentioned. We are actively resolving bugs in our [releases](https://github.com/prisma/prisma/releases). Can you reshare the bug you mentioned so I can share it with the team?

**Reply by Akumzy** [reply]
My bad, I don't understand how I linked the wrong issue but here it is [issue](https://github.com/prisma/prisma/issues/20586)

here is my version

```prisma
model Group {
  id         String @id @default(uuid())
uniqueName String @unique @default(dbgenerated("translate(substr(gen_random_uuid()::text, 1, 11), '-', '')")) @db.VarChar(255)
}
```

by adding that dbgenerated everything time we run `npx prisma migrate dev` a new schema will be generate for `uniueName`

```sql
-- AlterTable
ALTER TABLE "Group" ALTER COLUMN "uniqueName" SET DEFAULT translate(substr(gen_random_uuid()::text, 1, 11), '-', '');
```

It also happens when there was no change on our schema files

**Reply by ludralph** [reply]
Hi @Akumzy 

Thank you for sharing it. We have now introduced a process for how we want to manage issues going forward. You can please read about it [here](https://www.prisma.io/blog/prisma-orm-manifesto)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
